### ISSUE TITLE
Update Official:Translation Howto on the wiki

### ISSUE THREAD

**Issue by luzpaz** [scribusproject/scribus-translations#5]
Wiki is not working ATM so I'll post here:

- [ ] https://wiki.scribus.net/canvas/Translation_workflow  
- [ ] https://wiki.scribus.net/canvas/Official_talk:Translation_Howto

**Comment by luzpaz** [comment]
@gyuris please check my revision at https://wiki.scribus.net/canvas/Official_talk:Translation_Howto

**Comment by gyuris** [comment]
Excellent, I accept! Thanks.

**Comment by luzpaz** [comment]
Thanks :+1: 
Now we need to add a Transifex section: 
https://wiki.scribus.net/canvas/Official_talk:Translation_Howto#Add_Transifex_section_--_Kunda_.28talk.29_22:07.2C_2_November_2016_.28CET.29

**Comment by gyuris** [comment]
Yup, than I understand this: https://bugs.scribus.net/view.php?id=13860#c42288

**Comment by gyuris** [comment]
These are your previous screenshots: https://github.com/scribusproject/scribus/issues/15#issuecomment-119227578

**Comment by gyuris** [comment]
> 3) Screenshots of the process

Transifex has a very good own documentation with screenshots for [translators](http://docs.transifex.com/getting-started/translators/) and about the [web editor](http://docs.transifex.com/translation/translating-with-the-web-editor/), about [roles](http://docs.transifex.com/teams/understanding-user-roles/), for coordinators about [managing collaborators](http://docs.transifex.com/teams/managing-and-removing-collaborators/) and about [reviewing](http://docs.transifex.com/translation/reviewing-strings/), and more also about [offline translating](http://docs.transifex.com/translation/translating-offline/). I think we don't need to duplicate this. Just point translators to the official help.

> 4) Guidelines for Language Coordinators
> 5) Guidelines for Language Translators

Yes, they are necessary.

**Comment by luzpaz** [comment]
Can you identify what strings in the Qt framework specifically impact the issue you were running in to when you filed the bug for issue [13860[(https://bugs.scribus.net/view.php?id=13860) ? We can then tell the other translators to translate those strings.

**Comment by luzpaz** [comment]
@gyuris updated https://wiki.scribus.net/canvas/Official:Translation_Howto please check and provide feedback. Thanks!

**Comment by gyuris** [comment]
Related to [13860](https://bugs.scribus.net/view.php?id=13860): These strings were: "OK", "Cancel", "Close"... Nothing more difficult. Therefore wondered I why must we translate whole Qt framework. But we [did it](https://www.transifex.com/qt-hu/public/).

At first sight on [Wiki](https://wiki.scribus.net/canvas/Official:Translation_Howto): 

1. The link "check the list of languages that..." is broken. I can't edit "Official" pages, I can't fix it.
2. Why exists this very similar and official wiki page: https://wiki.scribus.net/canvas/Official:Translation_Update

To be continued...

**Comment by gyuris** [comment]
@luzpaz, this is my 1st proposal for guidelines for coordinators and translators. I know, linguistically isn't the prettiest. What do you think about?

# Guidelines for language coordinators

- Create your team. We recommend using [Transifex](https://www.transifex.com/) for teamwork. Teamwork is good for load balancing and pinpointing errors. Start blogging about Scribus translation, give news about your work on forums and social networking sites.
- Watch [join requests](https://www.transifex.com/scribus/teams/41672/requests/) on Transifex. Accept them or decline them.
- Don't make your volunteers waiting. The volunteers were motivated by fast and visible results. If they don't see results, or you don't deal with their work, they will move on or be passive.
- Prevent the intentional damage. Use Transifex's [review system](http://docs.transifex.com/translation/reviewing-strings/). The reviewed translations can not be modified with translation [role](http://docs.transifex.com/teams/understanding-user-roles/), practically they are blocked, translators can add only suggestions.
- Translate the [Scribus terminology dictionary](https://wiki.scribus.net/canvas/Glossary_of_technical_terms_used_in_Scribus_and_its_translations) in [Transifex glossary](https://www.transifex.com/scribus/scribus/glossary/en/), so translation will be uniform.
- If you start from scratch, organize days of the translation for the team, where they met with. It's always good for motivation. Motivation produce results.
- Check new contributions form you translators regularly. Use review system.
- Once you've reached 100%, please follow announcements about the Scribus project schedule on Transifex.

# Guidelines for translators

- Look for the given language translator free software community and know the translation guidelines, practice, common mistakes, terminology.
- Use Scribus. If you don't use it, you don't know it, and you can't translate it.
- Get to know the DTP terms. Translations of commercial DTP software (eg. InDesign) provide much help in this. Use glossary in Transifex.
- If you don't understand something and you can't translate, find the sting in the program user interface or in the source code.

**Comment by luzpaz** [comment]
@gyuris looks great

**Comment by luzpaz** [comment]
@MrB74 Please take a look and if possible give the OK for us to add this to the official Translation page. Thanks

**Comment by luzpaz** [comment]
I updated the content in https://github.com/scribusproject/scribus-translations/issues/5#issuecomment-261814143
How does it look?

**Comment by gyuris** [comment]
@luzpaz, thanks! It's better!
